### ISSUE TITLE
Accuracy of the prediction model

### ISSUE THREAD

**Issue by Moheeb77** [mdciri/YOLOv7-GRAZPEDWRI-DX#2]
I was testing the model that you produced and it is really great. Today I found some images in kaggle and tried to test the model again but I was surprised that it didn't produce a good result. could you test the image and check, I might be doing something wrong. 

![14](https://user-images.githubusercontent.com/10824158/227780679-4533815c-2d81-494e-86b6-29d1b42afd2a.png)
![14](https://user-images.githubusercontent.com/10824158/227780686-26a2d226-2e32-4860-97c9-5697a832ee06.jpg)

**Comment by mdciri** [comment]
I tried as well inference on such image and I did not get a good result. If I have to guess the reason why the model fails for this image is that:

- the GRAZPEDWRI-DX dataset contains bone fractures of patients in pediatric age, this looks like an adult hand;
- I am not sure because it passed a quite long time, but I think that the GRAZPEDWRI-DX dataset does not contain vertical or oblique fractures, but most transverse ones. So, if this type of fracture was not present in the training dataset, the model cannot predict it well.

may I know the Kaggle link where you got this image?
